src/ucs/datastruct/list.h

```diff
@@ -111,6 +111,7 @@ static inline void ucs_list_del(ucs_list_link_t *elem)
  */
 static inline int ucs_list_is_empty(const ucs_list_link_t *head)
 {
+    UCS_ASAN_ADDRESS_IS_VALID(head, sizeof(ucs_list_link_t));
```

Comment:
IMO we also need it in:
- ucs_list_length
- ucs_list_for_each
- ucs_list_for_each_safe
- ucs_list_is_first
- ucs_list_is_last